.gitignore

```diff
@@ -63,3 +63,6 @@ docs/html
 
 # pynev
 .python-version
+
+# MacOS
+.DS_Store
```

Comment:
:thinking:

Reply:
Strictly speaking, it isn't related to this PR, but still a good thing to have in the .gitignore if editing on a Mac

Reply:
Ok, removed it.